scss/main.scss

```diff
@@ -23,7 +23,8 @@ body {
 }
 
 .home {
-  background: linear-gradient($dk-btn-color, $lt-btn-color);
+  background: linear-gradient(#000000, #021338);
```

Comment:
Can we add these colours to the branding.scss and use the variable names here please 💃

Reply:
yeeeeees YEEEES

Reply:
Do we need to? it just random colors and wont be used again

Reply:
Yeee boi :D Gracias <3

Reply:
In `branding.scss` assign these to variables like:

```
// Gradient Colors
$black: #000000;
$dark-blue: ##021338;
```

Or something like that. branding is imported in the main css/scss files, so do a similar call like the one you took out!

Reply:
okay mams

Reply:
Yeah, I think it's good to add them to `branding.scss`. It'll be more consistent that way with how we're doing it and if we want to use the colors elsewhere they'll just be in that one file.

Reply:
i did it. it is in branding now